.github/workflows/pullrequest.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
     - run: npm run lint
     - run: npm run build --if-present
     - run: npm test
+    - run: docker build .
```

Comment:
Maybe we sperate the build job into a seperate workflow? This now mixes unit tests and building. But not blocking

Reply:
Sure I will do